prices/solana/coinpaprika.yaml

```diff
@@ -47,3 +47,18 @@
   symbol: USDT
   address: Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB
   decimals: 6
+- name: shdw-genesysgo-shadow
```

Comment:
this comment would apply to the name attribute on all entries, but i'm curious if we should give the name the cleaner look, similar to how it is on solscan?
https://solscan.io/token/SHDWyBxihqiCj6YekG2GUr7wqKLeLAMK1gHZck9pL6y
for this one, it'd be 'Shadow Token'

this is minor feedback, but curious your thoughts?

Reply:
good to know 👍